build/setup-import-controller.sh

```diff
@@ -170,7 +170,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: managedcluster-import-controller-agent-regitration-client
+  name: managedcluster-import-controller-agent-registration-client
```

Comment:
please test the upgrade case, if I remember correctly, the roleRef can not be updated.

Reply:
And for the change in the backplane operator https://github.com/stolostron/backplane-operator/pull/1077/files , why do we only need to update `ClusterRole`, how about the role binding, roleRef there?

Reply:
There is no rolebinding and roleRef for this clusterrole, the clusterrole is created for users who want to use agent-registration, they create their serviceaccount and clusterrolebinding. 

I put this clusterrole in backplane-operator only want to reduce a common clusterrole that users need to create. But users can create the clusterrole by themselves as well.

Reply:
We don't have roleRef need to updated, only this clusterrole.